README.md

```diff
@@ -8,7 +8,7 @@ Developers mailing list: https://dev.eclipse.org/mailman/listinfo/wakaama-dev
     -+- core                   (the LWM2M engine)
      |    |
      |    +- er-coap-13        (Slightly modified Erbium's CoAP engine from
```

Comment:
I just looked at the Contiki repository : https://github.com/contiki-os/contiki/tree/master/apps/er-coap

At first sight, the code seems more than "slightly modified" (I confess that my judgement is only based on number of files and files name, so it could be totally wrong  :sweat_smile:)

**Naive question**, this does not make sense to synchronize those 2 projects ? (I mean erbium in contiki and erbium in wakaama ?)

Reply:
At this point it might be a similar level of effort as #352 to update to the latest erbium code, but I haven't looked at it in detail. Why don't we create an issue to track that.

Reply:
> Why don't we create an issue to track that.

Yep, if you think it worth to look at,  that makes sense :)

Reply:
About "slightly modified" we let it like this even if this is maybe more than that ?
(This is not so important I suppose)

Reply:
Just checked out the [initial commit](https://github.com/eclipse/wakaama/commit/b1691d28c56b28ca1c317e79f365c3c282f8135e) of Wakaama, tried to find the matching sources in the [Contiki](https://github.com/contiki-os/contiki/) code base/history - I failed. It seems to me that Wakaama's copy of the code got changed even before the first commit.

I sent an email to the creator of the first commit, asking if there might still be a previous repository lying around somewhere. I guess this could provide us a conclusive answer.

Reply:
> About "slightly modified" we let it like this even if this is maybe more than that ?
> (This is not so important I suppose)

I removed the "slightly" in the statement - even if the changes introduced by Wakaama turn out to be small, the changes which in the Contiki project since leave us with quite a big delta, rendering the phrase misleading.

Reply:
@rettichschnidi before to be hosted by eclipse foundation, the project was host here : https://github.com/intel/liblwm2m

Reply:
@sbernard31 Created #495 using this information.